doc/sphinx/proof-engine/ltac.rst

```diff
@@ -252,6 +208,51 @@ improving performance.
         - a tactic
         -
 
+:n:`ltac:(@ltac_expr)` can be used to indicate that the parenthesized item
+should be interpreted as a tactic and not as a term.  The constructs can also
+be used to pass parameters to OCaml code.  (While all
```

Comment:
```suggestion
should be interpreted as a tactic and not as a term.  The constructs can also
be used to pass parameters to tactics written in OCaml.  (While all
```
(Otherwise it sounds like you can inject OCaml code directly in a .v file)